client/components/adminDashboard/PendingApprovals.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+import AdminSidebar from "./AdminSidebar";
+import PendingCard from "./PendingCard";
+
+class PendingApprovals extends Component {
+  constructor() {
+    super();
+    this.state = {
+      pendingStudentList: null
+    };
+  }
+
+  componentDidMount() {
+    fetch("/api/v1/admin/pending", {
```

Comment:
change the url to something more explicit. `/admin/students/pending`.

Reply:
Also, it should be part of redux action-creators

Reply:
Route changed. 

Pending: Implementation in redux.